.travis.yml

```diff
@@ -0,0 +1,37 @@
+# https://travis-ci.org/Anaconda-Platform/anaconda-client
+
+language: python
+sudo: false
+
+env:
+  global:
+    # Environment variables used by astropy helpers
+    - CONDA_CHANNELS="https://conda.anaconda.org/t/an-1126452d-8e3d-439a-9763-03b5696f4f97/binstar"
+    - CONDA_DEPENDENCIES="pip setuptools clyent pillow psutil nbformat anaconda_project"
```

Comment:
Another thing, to avoid the duplicate builds (since we only care about the merged state), should we add
```yaml
branches:
  only:
    - develop
```

to avoid the duplicate builds?